### DISCUSSION TITLE
ERROR with setup.java file

### DISCUSSION THREAD

**Discussion by t-sharabi** [apiman/apiman#2417] [Help] [answered]
Hello
I was trying to start the APIman using the docker but I am reaching to the point where jbang trying to staert "setup.java" and tried a lot of reading but no luck, so please if there is any one can help?

**terminal output**
_[admen@testvm apiman-docker-compose-3.0.0.Final]$ sudo docker-compose -f docker-compose.setup.yml up
Recreating apiman_init_1 ... done
Attaching to apiman_init_1
init_1  | jbang setup.java
init_1  | [jbang] [ERROR] Script or alias could not be found or read: 'setup.java'
init_1  | [jbang] Run with --verbose for more details
apiman_init_1 exited with code 2
[admen@testvm apiman-docker-compose-3.0.0.Final]$_

**Comment by msavy** [comment, marked as answer]
Please try without `sudo`.

**Reply by t-sharabi** [reply]
thanks, what I did is i have moved to root user

sudo -i
 then went through the steps and worked fine

thanks

**Comment by shekarnode** [comment]
I was facing the same issue on the windows system with rancher-desktop installed.
`For Windows, all files are automatically shared via WSL2.`
So you need to prefix your path `/mnt`
For example : `/mnt/d/apiman-kube/apiman-docker-compose-3.0.0.Final`

**Reply by msavy** [reply]
Is that not covered by the README? 

> TIP: If using Windows, set the working directory variable with `set PWD=%CD%` before running the steps below.

Or does additional information need to be added?

I don't use Windows, so not sure.

**Reply by shekarnode** [reply]
> Is that not covered by the README?
> 
> > TIP: If using Windows, set the working directory variable with `set PWD=%CD%` before running the steps below.
> 
> Or does additional information need to be added?
> 
> I don't use Windows, so not sure.

As I m using rancher-desktop(windows) for docker, so setting the env variable 'set PWD=%CD%' doesn't work as it won't return the correct path.
for Windows Subsystem for Linux filesystem, all the drives are present inside the `/mnt` directory.
